api/app/src/external/fhir/patient/index.ts

```diff
@@ -18,29 +18,36 @@ export const toFHIR = (patient: Patient): FHIRPatient => {
     identifier: convertDriversLicenseToIdentifier(patient.data.personalIdentifiers),
     name: [
       {
-        family: patient.data.lastName,
-        given: [patient.data.firstName],
+        family: patient.data.lastName[0],
+        given: patient.data.firstName,
       },
     ],
     telecom: patient.data.contact
-      ? Object.entries(patient.data.contact).map(([key, val]) => {
-          return {
-            system: key as ContactTypes,
-            value: val,
-          };
+      ? patient.data.contact.map(contact => {
+          return contact
+            ? Object.entries(contact).map(([key, val]) => {
+                return {
+                  system: key as ContactTypes,
+                  value: val ?? undefined,
+                };
+              })[0]
```

Comment:
we might be returning `email` for `telecom`... we should filter to make sure we're getting a element/record with `phone` set, right?

Reply:
`telecom` is a contact point that can be phone or email:

`A contact detail (e.g. a telephone number or an email address) by which the individual may be contacted.`